### PR TITLE
Commiting the test and the functionality file for Task 0

### PR DESCRIPTION
**Pull request by Farhanisto**

### CODE REVIEW

**Comment by Farhanisto** [comment]
Peter i am yet to add the people with rooms to the allocated list

**Comment by Farhanisto** [comment]
Thanks for the review i will check on this feedback

**Comment by Bopchy** [comment]
Your commit messages should be in imperative mood. See -- https://chris.beams.io/posts/git-commit/#imperative 

Also, your commit messages should be more descriptive, that is; instead of something like `changes made` you should have `change <the-specific-thing-you-changed>`. Your commits should also be more atomic - one commit should be focused on one thing so that you do not have to have messages like `added setUp()to test file and added the doc opt file`

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/venv
+*.py~
```
Comment (Bopchy) [inline .gitignore:2]: What does `~` mean here?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+*.pyc
+*.gitignore~
```
Comment (Bopchy) [inline .gitignore:4]: What does `~` mean here?

Comment (Farhanisto) [reply .gitignore:4]: There were this files with the extension .py~ created when ever i ran some a file so i added them here to disinclude them

**Pull request merged**